src/config/authd-config.c

```diff
@@ -104,7 +114,7 @@ int Read_Authd(XML_NODE node, void *d1, __attribute__((unused)) void *d2) {
                 return OS_INVALID;
             }
 
-            config->force_options.enabled = b;
+            config->force_options.enabled = (b ? true : false);
```

Comment:
The `eval_bool` already returns 1 or 0, and the macros `true` and `false` are in fact 1 and 0. 
Do we really need this `if` ?

Reply:
Fixed!